### PR TITLE
Added en_ud.lang

### PR DESCRIPTION
**Pull request by The-Fireplace**

### CODE REVIEW

**Comment by williewillus** [comment]
Is this actually a selectable locale in-game? TIL

**Comment by liach** [comment]
It is. I think a final solution is to add an upside-down localization generator.

**Comment by Drullkus** [comment]
screen shot 2018-02-17 at 10 28 34 pm

**Comment by xbony2** [comment]
👍

File: src/main/resources/assets/twilightforest/lang/en_ud.lang [hunk 1]
```diff
@@ -0,0 +1,622 @@
...
+twilightforest.config.loading_icon_scale=ᵷuᴉꞁɐɔS uoɔI
+twilightforest.config.loading_icon_scale.tooltip=˙uoɔᴉ ᵷuᴉpɐoꞁ ʎɔunoq ǝꞁoɥʍ ɟo ǝꞁɐɔS
+twilightforest.config.loading_icon_bounciness=ssǝuᴉɔunoᗺ uoɔI
+twilightforest.config.loading_icon_bounciness.tooltip=˙sǝɔunoq uoɔᴉ ᵷuᴉpɐoꞁ ǝɥʇ ɥɔnɯ ʍoH
+twilightforest.config.loading_icon_tilting=ssǝuᴉꞁqqoM uoɔI
+twilightforest.config.loading_icon_tilting.tooltip=˙sǝꞁqqoʍ uoɔᴉ ᵷuᴉpɐoꞁ ǝɥʇ ɹɐɟ ʍoH
+twilightforest.config.loading_icon_tilt_pushback=ʇǝsɟɟO ʇꞁᴉ⟘ uoɔI
+twilightforest.config.loading_icon_tilt_pushback.tooltip=˙uoɔᴉ ᵷuᴉpɐoꞁ ɟo ǝꞁqqoʍ ǝɥʇ ɹǝʇuǝɔ-ǝɹ oʇ ǝnꞁɐʌ ʞɔɐqɥsnԀ
+
+twilightforest.book.lichtower.1=ǝsɹnɔ ǝɥʇ ɯoɹɟ ɔᴉᵷɐɯ ǝɥ⟘ ˙ǝɹoɟǝq uǝǝs ǝʌ,I ʎuɐ uɐɥʇ ɹǝᵷuoɹʇs 'ǝsɹnɔ ɐ ʎq pǝʇɔǝʇoɹd ǝɹɐ ɹǝʍoʇ ǝɥʇ ɟo sʞɔᴉɹq ǝɥ⟘ ˙ɹǝʍoʇ sᴉɥʇ ᵷuᴉpunoɹɹns ɐɹnɐ ǝᵷuɐɹʇs ǝɥʇ ᵷuᴉuᴉɯɐxǝ unᵷǝq ǝʌɐɥ I\n\n0ⱢⱯ00\u[[sɹǝʇsuoɯ ʎq uo pǝʍɐuᵷ 'ʞooqǝʇou s,ɹǝɹoꞁdxǝ uⱯ]]8ⱢⱯ00\u
```
Comment (Tamaized) [inline src/main/resources/assets/twilightforest/lang/en_ud.lang:571]: pretty sure all of these are broken. Stuff like `\u00A78` got flipped as well as the order of some of the formatting codes

**Review by Tamaized — CHANGES_REQUESTED** [review]

**Comment by Drullkus** [comment]
Changes still requested, @The-Fireplace

**Comment by The-Fireplace** [comment]
I'll wait and re-do it when I get around to turning this into a gradle plugin.

**Pull request closed without merging**